File: aat-android/src/main/java/ch/bailu/aat/util/ui/ToolTipString.java
package ch.bailu.aat.util.ui;

import ch.bailu.aat_lib.util.ui.ToolTipProvider;

public class ToolTipString implements ToolTipProvider {

    private final String toolTip;

    public ToolTipString(String tt) {
        toolTip = tt;
    }

    @Override
    public String getToolTip() {
        return toolTip;
    }
}

File: aat-android/src/main/java/ch/bailu/aat/util/ui/tooltip/ToolTipString.kt
package ch.bailu.aat.util.ui.tooltip

import ch.bailu.aat_lib.util.ui.ToolTipProvider

class ToolTipString(private val toolTip: String) : ToolTipProvider {
    override fun getToolTip(): String {
        return toolTip
    }
}
